tests/system/robot/chromeTests.robot

```diff
@@ -40,3 +40,6 @@ ARIA treegrid
 	[Documentation]	Ensure that ARIA treegrids are accessible as a standard table in browse mode.
 	# Excluded due to regular failures.
 	test_ariaTreeGrid_browseMode
+ARIA checkbox
+	[Documentation]	Navigate to an unchecked checkbox in reading mode.

```

Comment:
**Note for future**: we may want to add a specific tag for the aria-at tests.
Although we may also want to consider some slight re-arrangement of the tests as well. I'd like the same tests to be run against FireFox also.

```suggestion
	[Documentation]	Navigate to an unchecked checkbox in reading mode.
	[Tags]	aria-at
```
See [tagging test cases.](https://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#tagging-test-cases) Then the `aria-at` tests [can be run specifically](https://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#by-tag-names) by including the argument `--include aria-at` when running the tests.